src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,53 @@
+package duke;
+
+import java.io.InputStreamReader;
+import java.util.Scanner;
+
+/**
+ * Represents the ui for the mainframe.
+ */
+public class Ui {
+final static String BINGUS_LOGO = " B   i   n   g   u   s ";
+    final static  String NAME = "Bingus";
+    /**
+     * Prints intro.
+     */
+    public static void intro() {
+        System.out.println("Hello from\n" + BINGUS_LOGO);
+        System.out.println("Hello! I'm \n" + NAME);
+        System.out.println("What can I do for you?\n");// initial introductory message
```

Comment:
It looks like you are repeating your comment. The javadocs for this method already states what the message does.

Same issue with methods such as `bye()`